src/main/java/com/mraof/minestuck/alchemy/GristHelper.java

```diff
@@ -135,58 +141,61 @@ public static void increase(Level level, PlayerIdentifier player, GristSet set)
 		data.setGristCache(newCache);
 	}
 	
-	public static void notify(MinecraftServer server, PlayerIdentifier player, GristSet set)
+	public static void notify(MinecraftServer server, PlayerIdentifier player, GristSet set, GristToast.EnumSource source, boolean increase)
```

Comment:
Basic javadoc on this function seems appropriate.

Reply:
Done. Feel free to tell me if you think it could use clarification.